src/Ar/A3BR/a3brControl.c

```diff
@@ -204,12 +216,17 @@ void A3brControl(struct A3brControl* inst){
 				brsstrcat( request.uri, "?action=stop&json=1");	
 				break;
 			case A3BR_API_VERSION_2:
-				brsstrcpy( request.uri, "/rw/rapid/execution/stop?mastership=implicit");
+				brsstrcpy( request.uri, "/rw/rapid/execution/stop");
+				brsstrcat( request.uri, "?json=1");	
 				break;
 		}
 		request.dataType = A3BR_REQ_DATA_TYPE_PARS;
 		request.errorCallback = &A3brControlErrorCallback;
 		request.successCallback = &A3brControlSuccessCallback;
+		brsstrcpy( request.parameters[0].name, "stopmode" );
+		brsstrcpy( request.parameters[0].value, "stop" );  //options - {cycle | instr | stop | qstop} 
+		brsstrcpy( request.parameters[1].name, "usetsp" );
+		brsstrcpy( request.parameters[1].value, "normal" );  //options - {normal | alltsk}
```

Comment:
It looks like these were missing for V1 as well, is that the case?

Reply:
It appears to be.  If parameters are not included in the http stop POST, then the default values will be used.  These values of the parameters are set to the default values (stop and normal) so there shouldn't be a change in a robot's expected reaction if it's using v1.0.